client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -371,196 +399,274 @@ export default function CampaignItemDetails( props: Props ) {
+													<InfoPopover
+														className="campaign-item-conversions__info-button"
+														position="bottom right"
+													>
+														{ __( 'Conversions:' ) }
+														<br />
+														<span className="popover-title">
+															{ __(
+																'show how many people made a purchase or completed a specific goal that aligns with the objectives of the campaign.'
+															) }
+														</span>
+													</InfoPopover>
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? conversionsTotalFormatted : <FlexibleSkeleton /> }
+												</span>
+											</div>
+											<div>
+												<span className="campaign-item-details__label">
+													{ translate( 'Conversion Rate' ) }
+													<InfoPopover
+														className="campaign-item-conversion-rate__info-button"
+														position="bottom right"
+													>
+														{ __( 'Conversion Rate:' ) }
+														<br />
+														<span className="popover-title">
+															{ __(
+																'shows the percentage of users who made a purchase (or completed a specific goal that aligns with the objectives of the campaign) out of the total number of users who clicked on the ad. Example: If your ad receives 100 clicks, and 5 people make a purchase, your conversion rate would be 5%.'
+															) }
+														</span>
+													</InfoPopover>
+												</span>
+												<span className="campaign-item-details__text wp-brand-font">
+													{ ! isLoading ? conversionsRateFormatted : <FlexibleSkeleton /> }
+												</span>
+											</div>
 										</div>
-									</div>
+									) }
 								</div>
-
+							</div>
+						) }
+						<div className="campaign-item-details__main-stats-container">
+							<div className="campaign-item-details__secondary-stats">
 								<div className="campaign-item-details__secondary-stats-row">
 									<div>
-										<span className="campaign-item-details__label">
-											{ translate( 'Audience' ) }
+										<span className="campaign-item-details__label">{ translate( 'Budget' ) }</span>
+										<span className="campaign-item-details__text wp-brand-font">
+											{ ! isLoading ? totalBudgetFormatted : <FlexibleSkeleton /> }
 										</span>
 										<span className="campaign-item-details__details">
-											{ ! isLoading ? devicesListFormatted : <FlexibleSkeleton /> }
-										</span>
-										<span className="campaign-item-details__label">
-											{ translate( 'Location' ) }
-										</span>
-										<span className="campaign-item-details__details campaign-item-details__locations">
 											{ ! isLoading ? (
-												<TargetLocations audienceList={ audience_list } />
+												/* translators: Daily average spend. dailyAverageSpending is the budget */
+												translate( 'Daily av. spend: $%(dailyAverageSpending)', {
```

Comment:
Similar to a previous comment, you need to specify `%s` instead of `%(dailyAverageSpending)`